components/automate-ui/src/app/helpers/auth/sorter.ts

```diff
@@ -0,0 +1,29 @@
+
+export class ChefSorters {
+
+    // normalSort takes in an Object and returns an Array of Objects
+    // sorted by the property chosen.
+
+    public static normalSort(input: object, property: string): Array<any> {
```

Comment:
Possibly more helpful to name it ObjectNormalSort for clarity.

Reply:
Step back, for a moment: what are you trying to convey with "normal" ? Not sure that adds value. I suspect you may want to make this `oneFieldSort` by the time you're done with the additional work.

Reply:
Since this is adding a natural sort behavior, is calling this `naturalSort` a good option?

Reply:
@phiggins would you mind helping me understand a little better how I would use a function in this case?

Reply:
I was imagining similar to how underscore's sortBy works: https://underscorejs.org/#sortBy

Kind of ironic that looking at the docs for that I can see it takes a function or the property name.

Reply:
thank you!  something to consider :)